src/Runner.php

```diff
@@ -44,7 +49,13 @@ public function __construct(Container $container = null)
      */
     public function run(InputInterface $input, OutputInterface $output)
     {
+        if (!empty($cassette = $this->config->get('vcr_cassette')) && !empty($mode = $this->config->get('vcr_mode'))) {
+            $this->startVCR(array_merge(compact('cassette'), compact('mode')));
```

Comment:
Moving the VCR-setting into the Behat feature context did not work. This does, though!